src/mscorlib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -777,6 +776,76 @@ public int EnsureCapacity(int capacity)
             return newSize;
         }
 
+        /// <summary>
+        /// Sets the capacity of this dictionary to what it would be if it had been originally initialized with all its entries
+        /// 
+        /// This method can be used to minimize the memory overhead 
+        /// once it is known that no new elements will be added. 
+        /// 
+        /// To completely clear a dictionary and release all memory 
+        /// referenced by the dictionary, execute the following statements:
+        /// 
+        /// dictionary.Clear();
+        /// dictionary.TrimExcess();
+        /// </summary>
+        public void TrimExcess()
+        {
+            TrimExcess(HashHelpers.GetPrime(Count));
```

Comment:
`GetPrime` does not need to be called here now that `TrimExcess(int capacity)` calls it.